desktop/core/src/desktop/api2.py

```diff
@@ -813,6 +814,44 @@ def export_documents(request):
     return make_response(f.getvalue(), 'json', filename)
 
 
+def topological_sort(docs):
+
```

Comment:
Suggestion: Add docstring for the method with input and return value description and why this method was required (py2 vs py3). This can help identify all py2 related stuff later for cleanup tasks.

Reply:
You mentioned https://code.djangoproject.com/ticket/26291 was fixed in Django 2.2. We are using Django 3.2.16. Your fix will be used in both Py2 and Py3?

Reply:
Yes @wing2fly , will be used in both the versions. 
although it will be redundant in python3 base Hue as we are using Django 3.2.16.

Reply:
Now we will use this code only for python2 based Hue